x-pack/plugins/uptime/public/components/monitor/monitor_status_details/__test__/monitor_redirects.test.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { renderWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { Ping } from '../../../../../common/runtime_types';
+import { MonitorRedirects } from '../monitor_status_bar/monitor_redirects';
+
+describe('MonitorRedirects component', () => {
+  let monitorStatus: Ping;
+
+  beforeEach(() => {
+    monitorStatus = {
```

Comment:
Two thoughts on this test code:

1. The functions we're testing take in way more data than they need (the whole `Ping`)
2. We're using ad-hoc inline fixtures here, which will be hard to update if / when the heartbeat schema changes.

I feel like if we adjust the functions to take a minimal datastructure we can decouple the test dep here. Thoughts?